content/guides/getting-started.md

```diff
@@ -109,7 +109,7 @@ __src/index.js__
 +
   function component() {
     var element = document.createElement('div');
-  
+
 -   // Lodash, currently included via a script, is required for this line to work
```

Comment:
Ah yeah sorry about that. I think I either misunderstood or took this out accidentally. Will remove this change tonight.